specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/definitions.json

```diff
@@ -5901,6 +5901,112 @@
         }
       }
+    },
+    "PortalRevisionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PortalRevisionContract"
+          },
+          "description": "Collection of portal revisions.",
+          "readOnly": true
+        },
+        "count": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Total record count number across all pages."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link, if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged list of portal revisions."
+    },
+    "PortalRevisionContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PortalRevisionContractProperties",
+          "description": "Properties of the portal revisions."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Portal revisions contract details."
+    },
+    "PortalRevisionContractProperties": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Portal revision identifier.",
+          "minLength": 1,
+          "maxLength": 256
+        },
+        "description": {
+          "type": "string",
+          "description": "Portal revision description.",
+          "maxLength": 2000
+        },
+        "statusDetails": {
+          "type": "string",
+          "description": "Portal revision publishing status details.",
+          "maxLength": 2000
+        },
+        "status": {
+          "type": "string",
+          "description": "Portal revision publishing status",
+          "enum": [
+            "pending",
+            "publishing",
+            "completed",
+            "failed"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "portalRevisionStatus",
+            "values": [
+              {
+                "value": "pending",
+                "description": "Portal revision publishing is pending"
+              },
+              {
+                "value": "publishing",
+                "description": "Portal revision is publishing"
+              },
+              {
+                "value": "completed",
+                "description": "Portal revision publishing completed"
+              },
+              {
+                "value": "failed",
+                "description": "Portal revision publishing failed"
+              }
+            ]
+          }
+        },
+        "isCurrent": {
+          "type": "boolean",
+          "description": "Indicates if the Portal Revision is public."
+        },
+        "createdDateTime": {
```

Comment:
Metadata around creation/updates should be placed in the systemData contract.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources  and https://armwiki.azurewebsites.net/api_contracts/ResourceSystemData.html?q=systemdata

Reply:
We want to do it consistently across all entities in the next API version. This API version is already released publicly.

Reply:
I'm ok with that.